starter-code/index.html

```diff
@@ -3,40 +3,90 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link href="./css/main.css" rel="stylesheet">
 </head>
 <body>
-Premium Discover Help Download
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
+<!-- nav bar -->
 
-What’s on Spotify?
+<div class="nav-bar">
+  <div class="logo"><img src="./images/spotify-logo.png" alt="logo"></div>
+<div class="links">
```

Comment:
Please start getting into the habit of properly spacing/tabbing your divs, because all of your divs are on the same left side as the body tag, so now it is hard to follow where starting and closing tags start and end.